app/views/analytics/index.html.erb

```diff
@@ -153,11 +153,45 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
+  <!-- Editors List-->
+    <div class="col-sm-4">
       <div class="chart-wrapper">
         <div class="chart-title background-orange">
+          Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+        </div>
+        <div class="chart-stage">
+         <table class='table leader-list'>
+           <tr>
+            <th></th>
+            <th>Editor</th>
+            <th>Updates</th>
+           </tr>
+
+           <% case_updaters_count(10).each_with_index do |who, i| %>
+            <tr>
+                <% unless who[0].to_i == 0 %>
```

Comment:
Can you explain what `who` is supposed to refer to?  At this point, can we just stop using this as a variable name, because this is not the first time where I've had a question of what this is supposed to be and why it's an array.

Reply:
`who` refers to each array returned by the helper method as it is looped through.

We can change the variable name.  What are you thinking?

I can also add a comment to explain.

Reply:
It might be better to, instead of doing something like this:

```
users.each do |who|
  logic with who[0]
  logic with who[1]
end
```

to instead do something like this:

```
users.each do |[user_id, count]|
  logic with user_id
  logic with count
end

```
That way you don't have to leave a comment.